x-pack/plugins/ml/public/ml_url_generator/dataframe_analytics_urls_generator.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/**
+ * Creates URL to the DataFrameAnalytics page
+ */
+import {
+  DataFrameAnalyticsExplorationQueryState,
+  DataFrameAnalyticsExplorationUrlState,
+  DataFrameAnalyticsQueryState,
+  DataFrameAnalyticsUrlState,
+} from '../../common/types/ml_url_generator';
+import { ML_PAGES } from '../../common/constants/ml_url_generator';
+import { setStateToKbnUrl } from '../../../../../src/plugins/kibana_utils/public';
+
+export function createDataframeAnalyticsUrl(
+  appBasePath: string,
+  mlUrlGeneratorState: DataFrameAnalyticsUrlState['pageState']
+): string {
+  let url = `${appBasePath}/${ML_PAGES.DATA_FRAME_ANALYTICS_JOBS_MANAGE}`;
+
+  if (mlUrlGeneratorState) {
+    const { jobId, groupIds } = mlUrlGeneratorState;
+    const queryState: Partial<DataFrameAnalyticsQueryState> = {
+      jobId,
+      groupIds,
+    };
+
+    url = setStateToKbnUrl<Partial<DataFrameAnalyticsQueryState>>(
+      'mlManagement',
+      queryState,
+      { useHash: false, storeInHashQuery: false },
+      url
+    );
+  }
+
+  return url;
+}
+
+/**
+ * Creates URL to the DataFrameAnalytics Exploration page
+ */
+export function createDataframeAnalyticsExplorationUrl(
```

Comment:
Again for consistency, should be `createDataFrameAnalyticsExplorationUrl`

Reply:
Renamed here https://github.com/elastic/kibana/pull/75696/commits/9962c25afba12c5d6ebc699f1134f010176b068f